src/behavior_generation_lecture_python/graph_search/a_star.py

```diff
@@ -131,6 +131,12 @@ def a_star(self, start: str, end: str) -> bool:
         :param end: Name of the end node.
         :return: True if shortest path found, False otherwise.
         """
+        try:
+            self.nodes_dict["K"]
```

Comment:
`start`?

Reply:
Yeah pushed the testing code :man_facepalming: